src/plugins/stepsform/stepsform.js

```diff
@@ -0,0 +1,190 @@
+		var eventTarget = wb.init( event, componentName, selector ),
+			elmId;
+
+		if ( eventTarget ) {
+			elmId = eventTarget.id;
+
+			// Ensure there is a unique id on the element
+			if ( !elmId ) {
+				elmId = componentName + "-id-" + idCount;
+				idCount += 1;
+				eventTarget.id = elmId;
+			}
+
+			// Only initialize the i18nText once
+			if ( !i18nText ) {
+				i18n = wb.i18n;
+				i18nText = {
+					prv: i18n( "prv" ),
+					nxt: i18n( "nxt" )
+				};
+			}
+
+			// remove submit button (user should add one for no script purposes)
+			$( eventTarget ).find( "input[type=submit]" ).remove();
+
+			// find all fieldsets, hide required content, and add appropriate buttons
+			var fieldsets = eventTarget.getElementsByTagName( "FIELDSET" ),
+				settings = wb.getData( eventTarget, componentName ),
+				form, fieldset, legend, div;
+			for ( var i = 0, len = fieldsets.length; i < len; i++ ) {
+
+				// only applies too fieldsets that have a parent element of form
+				fieldset = fieldsets[ i ];
+				form = fieldset.parentElement;
+				if ( form.nodeName.toUpperCase() === "FORM" ) {
+					var isFirstFieldset = ( i === 0 ) ? true : false,
+						isLastFieldset = ( ( i + 1 ) >= len ) ? true : false,
+						nextFieldsetParent = ( !isLastFieldset && fieldsets[ i + 1 ] ) ? fieldsets[ i + 1 ].parentElement : false,
+						previous, next, submit;
+
+					legend = fieldset.getElementsByTagName( "LEGEND" )[ 0 ];
+					div = fieldset.getElementsByTagName( "DIV" )[ 0 ];
+
+					// add previous step button
+					if ( !isFirstFieldset ) {
+						previous = createStepsControls( "button", true, "mrgn-rght-sm mrgn-bttm-md", i18nText.prv );
+						if ( previous ) {
+							previous.addEventListener( "click", function() {
+								displayFieldset( this.parentElement, false );
+							} );
+							div.appendChild( previous );
+						}
+					}
+
+					// add next step button or form submit button if last fieldset
+					if ( nextFieldsetParent && nextFieldsetParent.nodeName.toUpperCase() === "FORM" ) {
+						next = createStepsControls( "button", true, "mrgn-bttm-md", i18nText.nxt );
+						if ( next ) {
+							next.addEventListener( "click", function( evt ) {
+								evt.preventDefault(); // stop form submission
+
+								// confirm if form is valid
+								var isFormValid = true;
+								if ( jQuery.validator && jQuery.validator !== "undefined" ) {
+									isFormValid =  $( "#" + this.parentElement.parentElement.parentElement.id ).valid();
+									this.parentElement.previousElementSibling.setAttribute( "class", "wb-steps-error" );
+								}
+
+								// continue if valid
+								if ( isFormValid ) {
+									this.parentElement.previousElementSibling.removeAttribute( "class" );
+									displayFieldset( this.parentElement, true );
+								}
+
+							} );
+							div.appendChild( next );
+						}
+					} else {
+						var value = ( settings && settings.submitBtnVal ) ? settings.submitBtnVal : "X",
```

Comment:
- [ ] i18n Text (add "Submit" as default button text (enabling tmpl-submit (not sure about this)))